### ISSUE TITLE
Show a warning if a glossary entry isn't in a translation

### ISSUE THREAD

**Issue by ocean90** [GlotPress/GlotPress-WP#731]
The current glossary entries are only shown in a title attribute. I think we should also check a suggested translation against the glossary again and show a warning if a word that is supposed to be in a translation is not there.

**Comment by tobifjellner** [comment]
Good idea IF you allow the editor to also check a box whether this should be used or not for a specific entry. (And some bulk option to set or delete this setting for a whole bunch of entries).
There are phrases and words where you may have different possibilities to translate, including many situations where the best "translation" is to actually **not** translate a word.
One example: Depending on which class and number of noun is described, the English word _Invalid_ may have at least 3 different translations in Swedish (Ogiltig, Ogiltigt, Ogiltiga). In Russian it would be even more, depending on case, numerus and class...

**Comment by Mte90** [comment]
The last version of GlotDict implement that.
As I can see the problem is that with plural forms in the and terms in the glossary.
As example: 
* option -> opzione
* options -> opzioni

In the Italian glossary there is only the single form (option) but in the translation I search in the text for that translation and not on plural form.

**Comment by akirk** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add a warning if a glossary term is not used in the translation](https://trello.com/c/qq2w6Rr4/6-add-a-warning-if-a-glossary-term-is-not-used-in-the-translation)
